addons/website_sale_slides/static/src/xml/website_slides_unsubscribe.xml

```diff
@@ -4,10 +4,12 @@
     <t t-inherit="website_slides.SlideUnsubscribeDialog" t-inherit-mode="extension">
```

Comment:
@Julien00859 Oy! Looks better now :) Looks almost good to me!

I think the first sentence of your commit message was not finished ^^

I did not know of gap-..., nor fa-...x, good to know :)

Also, small meta format tips:
- we usually use format "Task-01234567" for task number at the end of commit messages.
- And usually, (no need to change here, but for next branches) we use version-module-name-trigram for the branch name, so here it would be master-website_slides-leave_course_flex-juc or something along those lines. Just small :socks: remarks en passant ^^

About target: as this is a pure visual fix, I guess master is fine :)

Once you have updated your commit message, you can green :) Cheers!

Reply:
Done thank you :)

Noted for the branch naming scheme!